coral-incremental/build.gradle

```diff
@@ -0,0 +1,14 @@
+dependencies {
+  compile project(':coral-hive')
+
+  testCompile(deps.'hive'.'hive-exec-core') {
+    exclude group: 'org.apache.avro', module: 'avro-tools'
+    // These exclusions are required to prevent duplicate classes since we include
+    // shaded jar above
+    exclude group: 'org.apache.calcite', module: 'calcite-core'
+    exclude group: 'org.apache.calcite', module: 'calcite-avatica'
```

Comment:
@ljfgem could you take a look?

Reply:
I think 
```
    exclude group: 'org.apache.avro', module: 'avro-tools'
    exclude group: 'org.apache.calcite', module: 'calcite-avatica'
```
are not necessary, and please modify the comment to be:
```
    // This exclusion is required to prevent duplicate classes since we include
    // shaded 'calcite-core' transitively
```
Other parts LGTM.